osu.Game/Screens/Play/HUD/JudgementCounter/JudgementCounterDisplay.cs

```diff
@@ -109,6 +110,14 @@ bool shouldShow(JudgementCounter counter)
             }
         }
 
+        private void updateFirstChildVisibility()
```

Comment:
not sure about this method, the alternative was checking in updatemode() whether showmax is toggled, this felt cleaner so i opted for it